app/Repositories/PostRepository.php

```diff
@@ -96,10 +96,15 @@ public function create(array $data, $signupId, $authenticatedUserRole = null)
         $isAdmin = auth()->user() && auth()->user()->role === 'admin';
         $hasAdminScope = in_array('admin', token()->scopes());
 
-        // Admin users may provide a source and status when uploading a post.
+        // Admin users may provide a source, status, and created_at when uploading a post.
         if ($isAdmin || $hasAdminScope) {
             $post->status = isset($data['status']) ? $data['status'] : 'pending';
             $post->source = isset($data['source']) ? $data['source'] : token()->client();
+
+            // If there is a created_at property, fill this in (e.g. if created_at is sent when creating a record with the importer app).
+            if (isset($data['created_at'])) {
+                $post->created_at = $data['created_at'];
```

Comment:
great call! Updated :) Is it ok to just do `date` in the validation?

Reply:
@chloealee ah super cool that since the post already exists and you are updating `created_at` (and not `updated_at`) you don't have to call `$user->save(['timestamps' => false]);` because it won't try to overwrite `created_at` since it's already been created!